vispy/visuals/mesh.py

```diff
@@ -405,6 +408,22 @@ def _update_data(self):
                 colors = self._color.rgba
         self.shared_program.vert['position'] = self._vertices
 
+        if self._cmap.texture_map_data is not None:
```

Comment:
@djhoese @jarno-ssec 
Just thinking if moving this block (L411-L426) into a dedicated function would make things bit more easy with respect to code duplication (used 4 times in different Visuals). 

Since all code in this block depends only on `self._cmap` we could put the function (eg. get_cmap_texture_lut or similar) into `vispy.color.colormap` and return the `Texture2D` object:

``` python
self._texture_LUT = get_cmap_texture_lut(self._cmap)
self.shared_program['texture2D_LUT'] = self._texture_LUT
```
Maybe the underscore variable isn't necessary at all.

Thoughts?

Reply:
Sounds good to me.

Reply:
Is it more PEP8 friendly to do `texture_lut`? Not a big deal either way.